### DISCUSSION TITLE
--inspect-brk rename

### DISCUSSION THREAD

**Discussion by timreichen** [denoland/deno#9343] [General]
I was wondering if the deno run option `--inspect-brk` should be renamed to `--inspect-break` since non of the other flags are abbreviated and it would be more expressive. I had to read the help description to see what brk means😅.
I think 13 over 15 chars does not make the difference. What do you think?

**Comment by kitsonk** [comment]
Why did you open up #9344 after this?  Could we try to keep conversations focused and avoid creating duplicates?
